src/guide/hotspot-development.md

```diff
@@ -1,5 +1,10 @@
 # HotSpot Development
 
+See [Working With Pull Requests](#working-with-pull-requests) for generic guidance and requirements around pushing changes. For the HotSpot codebase there are a few additional requirements:
+
+* Your change must have been approved by two Committers out of which at least one is also a Reviewer
```

Comment:
The original requirement, from the Oracle-internal wiki ("Hotspot Code Review Guidelines"), was "At least two reviewers (one of which is a Reviewer)".  The "two Committers" requirement suggested here (taken from the openjdk wiki) was a quiet change introduced when the openjdk wiki page was created based on the internal wiki.  In earlier (private) discussion this change was questioned.  We want to encourage thoughtful and knowledgeable reviews even from folks who are new or only lightly connected to the project and so don't have Committer status (yet), rather than effectively saying such reviews don't count for anything in the official process.

Reply:
I agree - please keep this as "two reviewers".

Reply:
Agreed and fixed.